athena/tests/integration_tests/eunomia/eval/reporting/helpers/generate_scenario_dataframes.py

```
@@ -0,0 +1,87 @@
+import pandas as pd
+from typing import Dict
+from eval.models.entities.scenario_result import ScenarioResult
+from eval.models.entities.scenario import Scenario
+
+
+def generate_scenario_dataframes(
```

Pylint (3.3.7)

[refactor] Too many local variables (21/15)

(R0914)